src/main/scala/tech/cryptonomic/conseil/tezos/repositories/TezosRepositories.scala

```diff
@@ -0,0 +1,190 @@
+package tech.cryptonomic.conseil.tezos.repositories
+
+import tech.cryptonomic.conseil.tezos.Fees
+import tech.cryptonomic.conseil.tezos.Fees.AverageFees
+import tech.cryptonomic.conseil.tezos.TezosTypes.{
+  Account,
+  AccountId,
+  Block,
+  BlockHash,
+  BlockReference,
+  BlockTagged,
+  ContractId,
+  Delegate,
+  PublicKeyHash,
+  Voting
+}
+
+/** Defines repository operations related to chain blocks
+  * Parametric on the returned effect type wrapper
+  */
+trait BlocksRepository[Eff[_]] {
```

Comment:
Similar question to the one before..What is the value of abstraction here? As far as I can see, only `SlickRepositories` seems to implement these traits. Why not keep it concrete until there is a need to abstract?